payjoin/src/input_type.rs

```diff
@@ -56,13 +56,17 @@ impl<'de> serde::Deserialize<'de> for InputType {
             let ty = match parts[0].strip_prefix("type=") {
                 Some("pubkey") => SegWitV0Type::Pubkey,
                 Some("script") => SegWitV0Type::Script,
-                _ => return Err(serde::de::Error::custom("invalid SegWitV0 type")),
+                _ => {
+                    return Err(serde::de::Error::custom("invalid SegWitV0 type"));
+                }
```

Comment:
like this one

Reply:
Sure 👍

Reply:
fyi ohttp: 0.5.1 makes KeyConfig [Clone](https://docs.rs/ohttp/0.5.0/ohttp/struct.KeyConfig.html#impl-Clone-for-KeyConfig), so you might want to depend on that in your pr